android/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.0-alpha09'
+        classpath 'com.android.tools.build:gradle:3.1.2'
```

Comment:
Why was this change necessary?  (Or was this automatically performed?)

Reply:
I think Android Studio was yelling at me / did it automatically